NitroxClient/MonoBehaviours/Gui/Chat/PlayerChatEntry.cs

```diff
@@ -14,17 +15,17 @@ class PlayerChatEntry : MonoBehaviour
         private bool chatEnabled = false;
         private string chatMessage = "";
         private PlayerChatManager chatManager;
-        
+
```

Comment:
I do like that your editor fixes all these unnecessary (trailing) whitespaces, and spaces between keywords and parentheses :+1:.